### PR TITLE
Minion id is visible in System Info box

### PR DESCRIPTION
**Pull request by ncounter**
See https://github.com/uyuni-project/uyuni/pull/883

### CODE REVIEW

File: modules/reference/pages/systems/system-details.adoc [hunk 1]
```diff
@@ -70,6 +70,9 @@ The IP address of the client.
 IPv6 Address:::
 The IPv6 address of the client.
 
+Minion Id:::
+If the client is a salt minion, the minion id value is shown.
```
Comment (Loquacity) [inline modules/reference/pages/systems/system-details.adoc]:
```suggestion
For salt clients, the minion identification value is shown.
```

Comment (Loquacity) [inline modules/reference/pages/systems/system-details.adoc]:
```suggestion
On salt clients only, shows the minion identification value.
```

Comment (ncounter) [reply modules/reference/pages/systems/system-details.adoc]: That's the difference outcome when *you* are a mother language and a great writer......but I don't!! :wink:

Comment (Loquacity) [reply modules/reference/pages/systems/system-details.adoc]: I still love your work, Dario <3

**Pull request merged**